src/transformers/onnx/features.py

```diff
@@ -71,6 +75,36 @@ class FeaturesManager:
             "question-answering",
             onnx_config_cls=CamembertOnnxConfig,
         ),
+        "big-bird": supported_features_mapping(
```

Comment:
Currently the tests do not check if this works (it will be added very soon), have you checked that the export was working fine for each of these features manually?

Reply:
No, did not test each feature, just put them all in for completeness. I'll wait for the official test drivers to be released since it sounds like they're almost ready.